src/App.js

```diff
@@ -1,19 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import GlobalHeader from './Components/GlobalHeader';
-import Card from './Components/Card';
+import Card from './Components/Card/Card';
```

Comment:
Create index.js file in Components/Card folder with 2 lines: import Card & export Card. So you could import your component this way:
```suggestion
import Card from './Components/Card';
```